ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HeaderValidation.hs

```diff
@@ -247,6 +258,7 @@ castHeaderEnvelopeError = \case
     UnexpectedBlockNo  expected actual   -> UnexpectedBlockNo  expected actual
     UnexpectedSlotNo   expected actual   -> UnexpectedSlotNo   expected actual
     UnexpectedPrevHash oldTip   prevHash -> UnexpectedPrevHash oldTip (castHash prevHash)
+    CheckpointMismatch bNo expected actual -> CheckpointMismatch bNo expected actual
```

Comment:
Boring style remark: Our [style guide](https://ouroboros-consensus.cardano.intersectmbo.org/docs/for-developers/StyleGuide) asks for aligned `->` :sob:

Reply:
Yeah, eventually, it would be great to use some automated formatter, but for now, when modifying existing code, I think it is easiest to just follow the style guide :sob: